Specs/Data/Models/moving-box/moving-box.gltf

```diff
@@ -0,0 +1,324 @@
+{
```

Comment:
All it does is animate the box to the side, which helped with testing if the animations were working. It should render the first frame, and then not render a few frames after.